src/SFA.DAS.ApprenticeFeedback.Database/Tables/ApprenticeFeedbackTarget.sql

```diff
@@ -42,4 +42,10 @@ CREATE NONCLUSTERED INDEX [IX_ApprenticeFeedbackTarget_Status]
     ON [dbo].[ApprenticeFeedbackTarget]	( [Status] )
 	INCLUDE ( [Id], [Withdrawn] , [IsTransfer] );
 
+GO
+
+CREATE NONCLUSTERED INDEX IX_ApprenticeFeedbackTarget_Eligibility
```

Comment:
@ChrisJWoodcock maybe very minor point but can you add square brackets to the index name so update to `[IX_ApprenticeFeedbackTarget_Eligibility]` might not be necessary but keeps it consistent